Applications/Browser.md

```diff
@@ -170,6 +199,8 @@ Use the **Area of Interest (AOI)** tool to draw a rectangular or polygonal area
 
 Use the ![location](_images/icons/Location.png) icon to mark a location and re-centre to the **Point of Interest(POI)**
 
```

Comment:
Please remove the empty line here and link the next sentence to the one before (no line break). If you click on the file preview here you are able to see that it might get otherwise rendered quite ugly.

![image](https://github.com/eu-cdse/documentation/assets/60705209/79f817c7-67ff-4d30-9800-dfc9f9e61a94)